### ISSUE TITLE
why nzbhydra.mv.db is so big ? 649 MB 

### ISSUE THREAD

**Issue by tigermanIT** [theotherp/nzbhydra2#866]
Please can you check why nzbhydra.mv.db keep getting bigger 

C:\nzbhydra2-5.1.8-windows\data\database\nzbhydra.mv.db

file size : 649 MB 

https://file.io/kpnuANf8RP7h

**Comment by theotherp** [comment]
That's honestly not that big compared to other users' databases.
Nothing I can do about it.

**Comment by tigermanIT** [comment]
but it keeps getting bigger
it's 717 MB right now 
from 649 to 717 MB in 1 hour

**Comment by tigermanIT** [comment]
739 MB in a couple of minutes !
would be great if you can install
nzbhydra2-5.1.8-windows
and reproduce

**Comment by theotherp** [comment]
Not to be too flippant about this but come back when it's a couple of
gigabytes.
Unfortunately this is a known problem with the database library.
When you restart hydra it should get smaller because then it will try to
compress it. You can also try enabling advanced options and increasing the
time for this (something with database compression or shutdown time).

**Comment by MicMillon** [comment]
... mine is 45G

was 46g for a couple of days, just restarted it, and it went from 46 to 45g..

i do care about the stats.. its actually the main reason i use it. i want it to help me decide which indexers are preforming the best, so i know which ones are worth renewing. if i delete the file, will it regenerate? i dont mind loosing search history, i just want the stats.. are they maintained in another file?

Program info
Version: 5.1.11
Container version: v5.1.11-ls110
Container release type: Release
Container author: linuxserver.io

running inside a docker, on an unraid server (Version: 6.12.4)

root@RIG:~# ls -ltrh /mnt/user/System/Docker/AppData/nzbhydra2/database/
total 46G
-rw-r--r-- 1 nobody users 22M Jan 19  2023 nzbhydra.trace.db.old
-rw-r--r-- 1 nobody users 13M Sep  5 08:11 nzbhydra.trace.db
-rw-r--r-- 1 nobody users 46G Sep 17 22:28 nzbhydra.mv.db
root@RIG:~# ls -ltrh /mnt/user/System/Docker/AppData/nzbhydra2/database/
total 46G
-rw-r--r-- 1 nobody users 22M Jan 19  2023 nzbhydra.trace.db.old
-rw-r--r-- 1 nobody users 13M Sep  5 08:11 nzbhydra.trace.db
-rw-r--r-- 1 nobody users 46G Sep 19 12:36 nzbhydra.mv.db
root@RIG:~#
root@RIG:~# ls -ltrh /mnt/user/System/Docker/AppData/nzbhydra2/database/
total 46G
-rw-r--r-- 1 nobody users 22M Jan 19  2023 nzbhydra.trace.db.old
-rw-r--r-- 1 nobody users 13M Sep  5 08:11 nzbhydra.trace.db
-rw-r--r-- 1 nobody users 45G Sep 19 12:36 nzbhydra.mv.db
root@RIG:~#

**Comment by theotherp** [comment]
Yes, the database will be recreated on the next start. But the stats are
created on the fly from the history.
Can you please upload your debug infos zip?

**Comment by MicMillon** [comment]
[nzbhydra-debuginfos-2023-09-19-15-10.zip](https://github.com/theotherp/nzbhydra2/files/12666015/nzbhydra-debuginfos-2023-09-19-15-10.zip)

Attached

**Comment by MicMillon** [comment]
Hey, any feedback from the attached debuginfo file?

i'd really like to downsize that database...

**Comment by theotherp** [comment]
Try creating a backup. Does that work? Is the file in there so big as well? If it's smaller just replace the existing one with the one from the backup while hydra is down. See if it stays smaller.

**Comment by MicMillon** [comment]
Hey, so i tried that.. created the backup. shutdown the container, extracted the 19g nzbhydra.mv.db file from the backup, replaced the 45g file in the /appdata/nzbhydra/database directory, and restarted the hydra container. it came up, with my data there. with the database sitting around 20gb... came back a day or two later, and now its back up to the same 45g as before...
